runtime/service/src/intTest/java/org/apache/polaris/service/it/RestCatalogKeycloakFileIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.it;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.TestProfile;
+import java.util.Map;
+import org.apache.polaris.service.it.env.ClientPrincipal;
+import org.apache.polaris.service.it.env.ManagementApi;
+import org.apache.polaris.service.it.env.PolarisClient;
+import org.apache.polaris.service.it.test.PolarisRestCatalogFileIntegrationTest;
+import org.apache.polaris.test.commons.keycloak.KeycloakAccess;
+import org.apache.polaris.test.commons.keycloak.KeycloakProfile;
+
+@QuarkusIntegrationTest
+@TestProfile(KeycloakProfile.class)
+public class RestCatalogKeycloakFileIT extends PolarisRestCatalogFileIntegrationTest {
+
+  KeycloakAccess keycloak;
+
+  @Override
+  protected ClientPrincipal createTestPrincipal(
+      PolarisClient client, String principalName, String principalRole) {
+    ClientPrincipal principal = super.createTestPrincipal(client, principalName, principalRole);
+    keycloak.createRole(principalRole);
+    keycloak.createUser(
+        principalName,
+        // Use the same password as the client secret
```

Comment:
Why? As for me, it detracts from the test's validity because now it is not obvious whether authentication happens locally (client secret) or via Keycloak :thinking:

Reply:
I guessed the password was not used outside, but that's also a reason for not using the same password, right? ;) Thx for changing!